### DISCUSSION TITLE
Internationalized domain name is used when I create filters via the logger

### DISCUSSION THREAD

**Discussion by Yuki2718** [uBlockOrigin/uBlock-issues#3187] [General] [answered]
I don't think this has been the default behavior. When was it changed, and can the changre be reverted? We don't use IDN for officail filters but use `xn-*`.

**Comment by gorhill** [comment, marked as answer]
uBO supports using IDN, and I think it's best to use out of respect of users, to allow them to read domain names in their own language.

**Reply by Yuki2718** [reply]
All right, not a big issue.
